### PR TITLE
docs: add documentation for configuration options

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Add documentation for configuration options

## Motivation and Context:

Create a page that we can point people to when they ask how to change default config values.

## Types of changes:

Docs

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [x] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: docs/CONFIG.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```js
+module.exports = {
+  features: {
+    networkSelection: true,
+    mainnetAutopilot: true,
+  },
+}
+```
+
+#### 3. Enable experemental Static Channel Backup Restore feature.
```
Comment (korhaliv) [inline docs/CONFIG.md]: experemental -> experimental

File: docs/CONFIG.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- **custom-environment-variables.json**  
+  Defines a mapping between environment variables and config values.
+
+### Config Options
+
+Available config options change from time to time so rather than list them all here we recommended that you inspect the file [`config/default.js`](../config/default.js) in order to learn what configuration options are available.
+
+## Examples
+
+Followinbg are some examples of configu overrides that you might find useful when developing Zap or making your own custom build to run.
```
Comment (korhaliv) [inline docs/CONFIG.md]: Followinbg -> Following
configu -> config

File: docs/CONFIG.md (hunk 1 above)
Comment (mrfelton) [reply docs/CONFIG.md]: updated

File: docs/CONFIG.md (hunk 2 above)
Comment (mrfelton) [reply docs/CONFIG.md]: updated

**Pull request merged**